### PR TITLE
Dodam teacher module

### LINKED ISSUE #158: Create Dodam Teacher Android/iOS Module
## Overview (Required)

- In fact, you need to set up a module that turns on the app to test and distribute various modules.

## Todo

- [x] Run iOS Simulator
- [x] Run Android Simulator
- [x] Set Dodam Teacher Module

### PR DESCRIPTION
**Pull request by 8954sood**
## Overview (Required)

-  For CMM, I tried to work with Xcode, but the module was still not recognized with the old version, so I solved it by uploading the Kotlin version and the rest of the module version.
- Create MultiplatformFeature, MultiplatformCompose Plugin
- Kotlin Version Up 1.9.22 To 2.0.0

## Issue

- close #158

### CODE REVIEW

**Review by wsi1212 — APPROVED** [review]
Dodam Teacher LGTM!! 🥰

File: iosApp/Configuration/Config.xcconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+TEAM_ID=
+BUNDLE_ID=com.b1nd.dodam.teacher.teacher
+APP_NAME=teacher
```
Comment (SeokgyuYun) [inline iosApp/Configuration/Config.xcconfig]: APP_NAME 수정해주세요.

File: iosApp/Configuration/Config.xcconfig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+TEAM_ID=
+BUNDLE_ID=com.b1nd.dodam.teacher.teacher
```
Comment (SeokgyuYun) [inline iosApp/Configuration/Config.xcconfig]: BUNDLE_ID에 teacher가 두번 들어가 있는데 com.b1nd.dodam.teacher로 수정해주세요.

**Review by SeokgyuYun — COMMENTED** [review]
iosApp을 dodam-teacher-ios로 변경하고 dodam-teacher를 dodam-teacher-android로 변경하면 구분하기 좋을 것 같아요.

**Comment by 8954sood** [comment]
> iosApp을 dodam-teacher-ios로 변경하고 dodam-teacher를 dodam-teacher-android로 변경하면 구분하기 좋을 것 같아요.

반영하였습니다.

**Pull request merged**